server/protocols/webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/WebAdminServerIntegrationTest.java

```diff
@@ -277,54 +277,6 @@ public void postShouldDoMigrationAndUpdateToTheLatestVersion() {
             .body(is("{\"version\":" + CassandraSchemaVersionManager.MAX_VERSION.getValue() + "}"));
     }
 
-    @Test
-    public void addressGroupsEndpointShouldHandleRequests() throws Exception {
```

Comment:
Why do we remove these tests that prooves that the routes are correctly binded? Can't we just write them using webadmin instead of the data-probe?